src/lib/mapping.js

```diff
@@ -70,7 +70,8 @@ function mapTeamList (row) {
     description: row.description,
     path: row.path,
     organizationId: row.org_id,
-    usersCount: parseInt(row.users_count)
+    usersCount: parseInt(row.users_count),
+    total: row.total
```

Comment:
My general feeling is: we should be consistent. So as long as we return the same structure for different paginated list I'm ok with that.

On the specific case: I think that field is the `totalUsersCount` @mihaidma used i his PR https://github.com/nearform/labs-authorization/pull/277/files#diff-c675678714399406311ee137b861e90fR425 . If it's so, I would use the same names/structure in all the "paginated endpoints"?